description: system ffmpeg build still relies on old ffmpeg_features label
author: chromium 65 authors

--- a/build/linux/unbundle/ffmpeg.gn
+++ b/build/linux/unbundle/ffmpeg.gn
@@ -14,8 +14,8 @@ pkg_config("system_ffmpeg") {
   ]
 }
 
-buildflag_header("ffmpeg_buildflags") {
-  header = "ffmpeg_buildflags.h"
+buildflag_header("ffmpeg_features") {
+  header = "ffmpeg_features.h"
   flags = [ "USE_SYSTEM_FFMPEG=true" ]
 }
 
@@ -30,7 +30,7 @@ shim_headers("ffmpeg_shim") {
 
 source_set("ffmpeg") {
   deps = [
-    ":ffmpeg_buildflags",
+    ":ffmpeg_features",
     ":ffmpeg_shim",
   ]
   public_configs = [ ":system_ffmpeg" ]
